src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue7045.cs

```diff
@@ -0,0 +1,24 @@
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues;
+
+public class Issue7045 : _IssuesUITest
+{
+	public Issue7045(TestDevice testDevice) : base(testDevice)
+	{
+	}
+
+	public override string Issue => "Shell BackButtonBehavior binding Command to valid ICommand causes back button to disappear";
+
+	[Test]
+	[Category(UITestCategories.Shell)]
+	public void ShellBackButtonBehaviorShouldWork()
+	{
+		App.WaitForElement("NavigateButton");
+		App.Click("NavigateButton");
+		App.WaitForElement("DetailPageLabel");
+		VerifyScreenshot();
```

Comment:
Pending snapshots. Already available in the latest build.

Example, Windows:
![image](https://github.com/user-attachments/assets/c66a79ac-f8d8-4b02-b313-e80a68a5d38d)

Could you commit the images?